website/docs/reference/widgets/table/column-settings.md

```diff
@@ -1,184 +1,165 @@
-# Column Settings
 
```

Comment:
Main comments for this page were:

- Remove Edit Table section
- Include important information from the current doc in Freeze column